DependencyInjection/Compiler/DbalSchemaFilterPass.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\ChildDefinition;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Processes the doctrine.dbal.schema_filter
+ */
+class DbalSchemaFilterPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        // feature require doctrine/dbal 2.9 or higher
+        if (!method_exists(\Doctrine\DBAL\Configuration::class, 'setSchemaAssetsFilter')) {
```

Comment:
Shouldn't we fail if the tag is used for older DBAL versions instead of silently ignoring the config ?